src/jit/gentree.cpp

```diff
@@ -5970,15 +5970,15 @@ GenTreeObj* Compiler::gtNewObjNode(CORINFO_CLASS_HANDLE structHnd, GenTree* addr
 // Creates a new CpObj node.
 // Parameters (exactly the same as MSIL CpObj):
 //
-//  dst       - The target to copy the struct to
-//  src       - The source to copy the struct from
-//  structHnd - A class token that represents the type of object being copied.
-//  volatil   - Is this marked as volatile memory?
-
+//  dst        - The target to copy the struct to
+//  src        - The source to copy the struct from
+//  structHnd  - A class token that represents the type of object being copied. May be null
+//               if FEATURE_SIMD is enabled and the source has a SIMD type.
+//  isVolatile - Is this marked as volatile memory?
```

Comment:
I would remove the question mark from this comment.
Possibly write:
 true if this is marked as a volatile operation